src/main/resources/application.properties

```diff
@@ -13,5 +13,5 @@ app.sendgrid.key:
 app.sendgrid.from-address:
 #JWT
 jwt.secret = 
-jwt.authorization = 
```

Comment:
added variable the enviroment, use this format ${ENVIROMENT NAME}

Reply:
Thanks, added variable the enviroment.